src/scripts/configure_npm.sh

```diff
@@ -16,7 +16,7 @@ then
   exit 1
 fi
 
-npm config set registry="$CLOUDSMITH_NPM_REGISTRY"
-npm config set "$CLOUDSMITH_NPM_AUTH_CONFIG"
+npm config set audit false registry="$CLOUDSMITH_NPM_REGISTRY"
+npm config set audit false "$CLOUDSMITH_NPM_AUTH_CONFIG"
```

Comment:
```suggestion
npm config set registry="$CLOUDSMITH_NPM_REGISTRY"
npm config set "$CLOUDSMITH_NPM_AUTH_CONFIG"
npm config set audit false
```

Think this needs to be split into a separate command.